code/model/SiteTreeFileExtension.php

```diff
@@ -88,31 +95,35 @@ public function BackLinkTrackingCount() {
 	}
 	
 	/**
-	 * Updates link tracking.
+	 * Updates link tracking in the current stage.
 	 */
 	public function onAfterDelete() {
+		// Skip live stage
```

Comment:
Because BackLinkTracking() isn't versioned, we only process back links on stage.